llama-index-integrations/llms/llama-index-llms-bedrock/llama_index/llms/bedrock/utils.py

```diff
@@ -107,18 +109,52 @@ def get_text_from_response(self, response: dict) -> str:
 
 
 def completion_to_anthopic_prompt(completion: str) -> str:
-    return messages_to_anthropic_prompt(prompt_to_messages(completion))
+    return messages_to_anthropic_messages(prompt_to_messages(completion))
+
+
+def _messages_to_anthropic_messages(messages: Sequence[ChatMessage]) -> List[dict]:
+    messages, system_prompt = messages_to_anthropic_messages(messages)
+    if system_prompt:
+        messages = [{"role": "system", "content": system_prompt}, *messages]
```

Comment:
```suggestion
    messages, _ = messages_to_anthropic_messages(messages)
```

Reply:
won't this change ignore the system prompt now though? `get_request_body` is handling when the system message is first.

Reply:
Anthropic only supports 2 roles in the chat messages. For passing the system prompt you need to pass it into the specialized key not in the message sequence.

Reply:
Yeah, just tested.

```log
E           botocore.errorfactory.ValidationException: An error occurred (ValidationException) when calling the InvokeModel operation: messages: roles must alternate between "user" and "assistant", but found multiple "user" roles in a row
```

Reply:
OK I figured out what was happening. It was an issue with my test case.

If the system prompt is sandwiched between user prompts you would end up would consecutive user prompts after transformation, which anthropic does not like at all. IMO this case should be handled in the anthropic utils.

Reply:
At least in the framework, I don't think it's common to have a system prompt sandwiched 

But definitely would be nice to handle that anyways. Maybe in a future PR?

From my view, I think this PR should be working for the most part right?

Reply:
Yes that can be scoped to different PR. Let's resolve this asap.